gcloud-java-dns/src/main/java/com/google/cloud/dns/testing/LocalDnsHelper.java

```diff
@@ -328,6 +352,71 @@ public void handle(HttpExchange exchange) throws IOException {
           requestMethod, exchange.getRequestURI())));
     }
 
+    private Response handleBatch(final HttpExchange exchange) throws IOException {
+      String contentType = exchange.getRequestHeaders().getFirst("Content-type");
+      if (contentType != null) {
+        int port = server.getAddress().getPort();
+        String responseBoundary = "____THIS_IS_HELPERS_BOUNDARY____";
```

Comment:
Do you think it'd be preferable to define this and the two vars below as private static final vars?  That way they won't be recreated each time a new batch request comes in.